src/dev/build/tasks/os_packages/package_scripts/postinst_upgrade.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+if command -v systemctl >/dev/null ; then
+    systemctl reload kibana.service
```

Comment:
This apparently doesn't do what I wanted *sigh*.

I'll change it to `systemctl daemon-reload`

Reply:
Good one here, it escaped to me! We want to reload the systemd manager configuration here and the correct is `systemctl daemon-reload`.